composite_controller.go

```diff
@@ -35,14 +34,14 @@ import (
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 )
 
-func syncAllCompositeControllers(dynClient *dynamicClientset, metaClient *internalclient.Clientset) error {
-	ccList, err := metaClient.MetacontrollerV1alpha1().CompositeControllers("").List(metav1.ListOptions{})
+func syncAllCompositeControllers(dynClient *dynamicClientset, mc *metaController) error {
+	ccList, err := (mc.ccLister).List(labels.NewSelector())
```

Comment:
We've seen bugs before caused by the ambiguity of whether the zero-value of `Selector` means "select everything" or "select nothing". Let' use `labels.Everything()` here so it's clear what we mean. Same for `icList` below.